stdlib/LinearAlgebra/src/lbt.jl

```diff
@@ -171,9 +171,20 @@ function Base.show(io::IO, mime::MIME{Symbol("text/plain")}, lbt::LBTConfig)
     end
 end
 
+# In the event that users want to call `lbt_get_config()` multiple times (e.g. for
+# runtime checks of which BLAS vendor is providing a symbol), let's cache the value
+# and clear it only when someone calls something that would cause it to change.
+const _cached_config = Ref{Union{Nothing,LBTConfig}}(nothing)
```

Comment:
If we want to make it "thead-safe", we might need the double-checked locking pattern or something here. That's tedious to do on a case-by-case basis so I guess we'd want it in Base.Threads.

Reply:
See #44387